src/platform/Ameba/Logging.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <platform_stdlib.h>
+
+namespace chip {
+namespace Logging {
+namespace Ameba {
+
+enum class AmebaLogLevel
+{
+    kError,
+    kProgress,
+    kDetail,
+};
+
+void LogSetLevel(uint8_t level);
```

Comment:
set and get should use `LogLevel` as type instead of `uint8_t`

Reply:
fixed